### PR TITLE
feat: activity feed quick wins

### PR DESCRIPTION
**Pull request by shirleyodeng**
## Summary

This PR;
- removes the leaderboard from the home page
- removes `All` and `Sponsors` from the activity filter dropdown
- shows activities of connections only

## Notion link

https://www.notion.so/talentprotocol/Activity-Feed-Quick-win-44574971ddd74ab8965a318b7541c47f

### CODE REVIEW

File: app/packs/src/components-v2/activity-wall/index.jsx [hunk 1]
```diff
@@ -7,11 +7,9 @@ import { Activities } from "src/components-v2/activities";
 const perPage = 8;
 
 const DROPDOWN_OPTIONS = [
-  { type: undefined, value: "All" },
   { type: ["Activities::GoalCreate", "Activities::GoalUpdate"], value: "Goals" },
   { type: "Activities::CareerUpdate", value: "Career Updates" },
-  { type: "Activities::ProfileComplete", value: "New Members" },
-  { type: "Activities::Sponsor", value: "Sponsors" }
+  { type: "Activities::ProfileComplete", value: "New Members" }
```
Comment (shirleyodeng) [inline app/packs/src/components-v2/activity-wall/index.jsx:12]: @RubenSousaDinis - is the "New Members" filter staying?

File: app/models/activity_feed.rb [hunk 2]
```diff
@@ -1,24 +1,18 @@
...
+      Activity
+        .where(global: true, origin_user_id: user.connections.pluck(:connected_user_id), type: types)
+        .to_sql
+
     user_activities = activities.where(type: types).to_sql
 
-    # Merge global and user-specific activities using UNION
-    Activity.from("(#{global_activities} UNION #{user_activities}) AS activities")
+    # Merge connection and user-specific activities using UNION
+    Activity.from("(#{connection_activities} UNION #{user_activities}) AS activities")
```
Comment (shirleyodeng) [inline app/models/activity_feed.rb:16]: @RubenSousaDinis - should we continue to show the user's own activities? Or only connections?

Comment (RubenSousaDinis) [reply app/models/activity_feed.rb:16]: own and from connections

File: app/packs/src/components-v2/activity-wall/index.jsx (hunk 1 above)
Comment (RubenSousaDinis) [reply app/packs/src/components-v2/activity-wall/index.jsx:12]: let's keep it for now

**Review by RubenSousaDinis — APPROVED** [review]

**Pull request merged**